charts/fluent-bit/aws-marketplace/README.md

```diff
@@ -0,0 +1,30 @@
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.  Please refer to
+Helm's [documentation](https://helm.sh/docs) to get started.
+
+Once Helm has been set up correctly, add the repo as follows:
+
+helm repo add calyptia https://calyptia.github.io/charts
+
+If you had already added this repo earlier, run `helm repo update` to retrieve
+the latest versions of the packages.  You can then run `helm search repo
+<alias>` to see the charts.
+
+To install the calyptia-fluentbit chart:
+
+    helm install calyptia-fluentbit <alias>/<chart-name>
+
+To uninstall the chart:
+
+    helm delete calyptia-fluentbit
+
+For AWS marketplace we need to set up a service account with the correct IAM roles:
+
+```
+eksctl create cluster
+
+eksctl utils associate-iam-oidc-provider --cluster=<clusterName> --approve
+
+eksctl create iamserviceaccount --cluster=<clusterName> --name=<serviceAccountName> --namespace=<serviceAccountNamespace> --attach-policy-arn=arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage --approve
```

Comment:
Do we need to provide this name to the chart?
Might be good to have a default name used here and in the chart.